### PR TITLE
Split auto pilot and object detector screens

### PR DESCRIPTION
**Pull request by dhruv2295**

### CODE REVIEW

File: android/app/src/main/java/org/openbot/main/MainFragment.java [hunk 1]
```diff
@@ -70,10 +71,17 @@ public void onItemClick(SubCategory subCategory) {
       case FeatureList.AI_MODELS:
```
Comment (thias15) [inline android/app/src/main/java/org/openbot/main/MainFragment.java:71]: Please refactor to autopilot

File: android/app/src/main/java/org/openbot/common/FeatureList.java [hunk 2]
```diff
@@ -71,7 +71,8 @@
     subCategories = new ArrayList<>();
     subCategories.add(new SubCategory(FREE_ROAM, R.drawable.ic_game, "#FFFF6D00"));
     subCategories.add(new SubCategory(DATA_COLLECTION, R.drawable.ic_storage, "#93C47D"));
-    subCategories.add(new SubCategory(AI_MODELS, R.drawable.ic_person_search, "#FFD966"));
+    subCategories.add(new SubCategory(AI_MODELS, R.drawable.ic_person_search, "#4B7BFF"));
```
Comment (thias15) [inline android/app/src/main/java/org/openbot/common/FeatureList.java:74]: refactor to autopilot and find a suitable icon

File: android/app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -109,4 +119,6 @@
     <string name="preview_resolution">Preview Resolution</string>
     <string name="file_available_body">A model with the same name already exists. Would you like to replace it?</string>
     <string name="file_available_title">File found</string>
+    <!-- TODO: Remove or change this placeholder text -->
+    <string name="hello_blank_fragment">Hello blank fragment</string>
```
Comment (thias15) [inline android/app/src/main/res/values/strings.xml:123]: what is this for?

**Review by thias15 — COMMENTED** [review]
Please address comments above and resolve conflicts.

File: android/app/src/main/res/values/strings.xml (hunk 3 above)
Comment (dhruv2295) [reply android/app/src/main/res/values/strings.xml:123]: nothing. just a dummy string android studio created. removed it.

File: android/app/src/main/java/org/openbot/main/MainFragment.java (hunk 1 above)
Comment (dhruv2295) [reply android/app/src/main/java/org/openbot/main/MainFragment.java:71]: done

File: android/app/src/main/java/org/openbot/common/FeatureList.java (hunk 2 above)
Comment (dhruv2295) [reply android/app/src/main/java/org/openbot/common/FeatureList.java]: done. not sure about the icon though. inputs welcome

**Comment by thias15** [comment]
Have encountered this issue a few times. 
`2021-04-17 17:24:52.039 26045-26045/org.openbot E/openbot: CameraFragment: Use case binding failed: java.lang.NullPointerException: Attempt to invoke virtual method 'void androidx.camera.lifecycle.ProcessCameraProvider.unbindAll()' on a null object reference
`

**Pull request merged**